airflow/models.py

```diff
@@ -2441,8 +2443,7 @@ class DagModel(Base):
     last_scheduler_run = Column(DateTime)
     # Last time this DAG was pickled
     last_pickled = Column(DateTime)
-    # When the DAG received a refreshed signal last, used to know when
-    # we need to force refresh
+    # The last time someone clicked on refresh
```

Comment:
Nit: # When the DAG last received a refresh signal (e.g. the DAG's "refresh" was clicked in the web UI)